core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly

```diff
@@ -26,6 +26,240 @@ THE SOFTWARE.
   User self sign up page.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <local:_entryFormPage host="${app}" title="${%Sign up}" action="createAccount" captcha="${it.isEnableCaptcha()}" xmlns:local="/hudson/security/HudsonPrivateSecurityRealm" />
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:x="jelly:xml" xmlns:f="/lib/form">
+
+    <st:setHeader name="Expires" value="0"/>
+    <st:setHeader name="Cache-Control" value="no-cache,no-store,must-revalidate"/>
+    <j:new var="h" className="hudson.Functions"/><!-- instead of JSP functions -->
+    <!-- response contentType header -->
+    <st:contentType value="text/html;charset=UTF-8"/>
+    <!--
+    needed to generate a session if non exists,
+    without it we would add ";jsessionid=" to the url which will result in a 404
+  -->
+    <j:set var="_" value="${request.getSession()}"/>
+    <!-- get default/commong page variable -->
+    ${h.initPageVariables(context)}
+    <x:doctype name="html"/>
+    <!-- in case of error we want to surround the form elements with an error hint -->
+    <j:set var="inputClass" value="${data.errorMessage!=null ? 'danger' : 'normal'}"/>
+    <j:set var="simpleDecorator" value="${h.simplePageDecorator}"/>
+    <html>
+        <head data-rooturl="${rootURL}" data-resurl="${resURL}" resURL="${resURL}">
+            <title>${h.appendIfNotNull('Create an account!', ' [Jenkins]', 'Jenkins')}</title>
+            <!-- we do not want bots on this page -->
+            <meta name="ROBOTS" content="NOFOLLOW"/>
+            <!-- css styling, will fallback to default implementation -->
+            <st:include it="${simpleDecorator}" page="simple-head.jelly" optional="true"/>
+            <link rel="stylesheet" href="${resURL}/css/signup.css" type="text/css"/>
+        </head>
+        <body>
+            <div class="simple-page">
+                <div class="modal signup">
+                    <div class="signupIntroDefault">
+                        <h1>${%Create an account!}</h1>
+                        <div>If you already have a Jenkins account,
+                            <a href="${rootURL}/login">please sign in</a>
```

Comment:
No trailing period is inconsistent with the `signUp` message on the sister screen.